docs/reference/routing.rst

```diff
@@ -130,10 +130,10 @@ the admin variable's ``generateUrl()`` command:
     <a href="{{ admin.generateUrl('list', params|merge('page': 1)) }}">List</a>
 
 Note that you do not need to provide the Admin's route prefix (``baseRouteName``) to
-generate a URL for the current Admin, just the action name.
+generate a URL for the current Admin, only the action name is needed.
 
-To generate a URL for a different Admin you just use the Route Name with the usual
-Twig helpers:
+To generate a URL for a different Admin you only need to use the Route Name with the
```

Comment:
this is still trying to say "its easy", it is still belittling language. could we say 
```
To generate a URL to a different Admin, call the Symfony Twig function ``path`` with the Route Name:
```

Reply:
Ups, fixed here: https://github.com/sonata-project/SonataAdminBundle/pull/5456